test/cypress/support/commands.ts

```diff
@@ -316,6 +323,12 @@ const commands = {
         return cy
             .exec(`./node_modules/.bin/babel --config-file ${babelConfigPath} ${scriptPath}`)
             .then(commandResult => commandResult.stdout);
+    },
+
+    disableGettingStarted: () => {
+        cy.interceptSp('GET', `/users/`, req => {
+            req.reply({ show_getting_started: false });
+        });
```

Comment:
Optional:

I suggest using a bit more declarative and IMO simpler approach:

```suggestion
        cy.interceptSp('GET', `/users/`, {
            body: { show_getting_started: false }
        });
```

Reply:
fixed